src/common/grid/example/index.jsx

```diff
@@ -0,0 +1,60 @@
+const Grid = FocusComponents.common.grid;
+const Column = FocusComponents.common.column;
+
+const GreySquare = React.createClass({
+    render() {
+        const label = this.props.label;
```

Comment:
@GuenoleK  You could have done a pure function here `const GreySquare = ({label}) => <div style={{backgroundColor: '#BDBDBD', height: '200px', color: 'white'}}>{label}</div>`